packages/admin/cms-admin/src/contentScope/noContentScopeError/NoContentScopeError.tsx

```diff
@@ -0,0 +1,15 @@
+import { type FunctionComponent } from "react";
+
+import { useCurrentUser } from "../../userPermissions/hooks/currentUser";
+import { StopImpersonationButton } from "../../userPermissions/user/ImpersonationButtons";
+
+export const NoContentScopeError: FunctionComponent = () => {
+    const user = useCurrentUser();
+
+    return (
+        <>
+            Error: user does not have access to any scopes.
```

Comment:
Somehow, translations do not lint well in the comet/cms-admin package. 

Lets look into this problem with: https://vivid-planet.atlassian.net/browse/COM-2210